interchain-exchange/tutorial/00-tutorial.md

```diff
@@ -16,23 +16,42 @@ If you want to see the end result, please refer to the [example implementation](
 - Send IBC packets from one blockchain to another
 - Deal with timeouts and acknowledgements of IBC packets
 
+## Concepts good to know
+
+**Order Books**
+
+**Order books** are electronic list of *buy* and *sell* orders for specific financial instrument - For eg: cryptocurrencies.
+An order book lists the number of tokens being *bid* (buy) on or *offered* (sell) at each price point, or market depth. It also identifies the market participants behind the buy and sell orders. These lists help traders and also improve market transparency because they provide valuable trading information.
+
+**Sell Order**
+
+**Sell order** enables a broker (blockchain) to sell tokens. A sell order may take any of a number of forms. Depending on the nature of the order, the broker (blockchain) may execute it at the best available price when the order is made, at a set price designated by the client (user), or according to a more complicated formula.
+
+**Buy Order**
+
+**Buy order** is an instruction from an investor (client / user) to a broker (blockchain) to buy a certain amount of tokens. Buy orders may take various forms. For example, an investor (client / user) may instruct the broker (blockchain) to buy immediately at the best available price, or to wait until a certain price is reached.
+
 ## How the module works
 
 You will learn how to build an exchange that works with two or more blockchains. The module is called `ibcdex`.
 
-The module allows to open an exchange order book between a pair of token from one blockchain and a token on another blockchain. The blockchains are required to have the `ibcdex` module available.
+The module allows to open an exchange order book between a pair of token within a blockchain network and between a pair of token on another blockchain network. The blockchains are required to have the `ibcdex` module available.
 
 Tokens can be bought or sold with Limit Orders on a simple order book, there is no notion of Liquidity Pool or AMM.
 
 The market is unidirectional: the token sold on the source chain cannot be bought back as it is, and the token bought from the target chain cannot be sold back using the same pair. If a token on a source chain is sold, it can only be bought back by creating a new pair on the order book. This is due to the nature of IBC, creating a `voucher` token on the target blockchain. In this tutorial you will learn the difference of a native blockchain token and a `voucher` token that is minted on another blockchain. You will learn how to create a second order book pair in order to receive the native token back.
 
+<!-- The above comment on line 27 is a bit hard to follow. Adding an example below for easier understanding.-->
```

Comment:
```suggestion
```

Comments are visible to users, let's not add them, unless they are meant for end-users.